setup.py

```diff
@@ -20,46 +20,46 @@
     'isort>=4.3',
     'pydocstyle>=3.0.0',
     'pytest-cov>=2.7.1',
-    'pytest-invenio>=1.2.1,<1.3.0',
+    'pytest-invenio>=1.3.1,<1.4.0',
     'pytest-pep8>=1.0.6',
     'pytest>=4.6.4,<5.0.0',
 ]
 
-db_version = '>=1.0.4,<1.1.0'
-search_version = '>=1.2.3,<1.3.0'
+db_version = '>=1.0.5,<1.1.0'
+search_version = '>=1.3.1,<1.4.0'
 
 extras_require = {
     # Bundles
     'base': [
-        'invenio-admin>=1.1.2,<1.2.0',
+        'invenio-admin>=1.2.0,<1.3.0',
         'invenio-assets>=1.1.3,<1.2.0',
```

Comment:
comment: it is okay, keep >=1.1.3,<1.2.0. must not be bumped, invenio-assets 1.2.0+ contains semantic-ui work.